lenskit-specs/src/main/java/org/lenskit/specs/data/TextDataSourceSpec.java

```diff
@@ -94,7 +111,7 @@ public void setBuilderType(String type) {
      * @param fields The list of fields.  Can be `null` to use the builder's default fields.
      */
     public void setFields(@Nullable List<String> fields) {
-        this.fields = fields != null ? new ArrayList<>(fields) : null;
+        this.fields = fields != null ? new ArrayList<String>(fields) : null;
```

Comment:
This change is not needed. Make sure that IntelliJ is set up to use the Java 7 language level in your Project Structure.